payments-model/src/main/java/com/stripe/android/repository/ConsumersApiService.kt

```diff
@@ -434,6 +471,11 @@ class ConsumersApiServiceImpl(
          */
         private val createPaymentDetails: String = getApiUrl("consumers/payment_details")
 
+        /**
+         * @return `https://api.stripe.com/v1/consumers/link_account_sessions`
+         */
+        private val createLinkAccountSession: String = getApiUrl("consumers/link_account_sessions")
+
```

Comment:
noticed that this exists in StripeApiRepository: https://github.com/stripe/stripe-android/blob/master/payments-core/src/main/java/com/stripe/android/networking/StripeApiRepository.kt#L1950-L1955

We should probably remove it from there (it's unused)